src/sentry/models/sourcemaperror.py

```diff
@@ -0,0 +1,29 @@
+"""
+A class for tracking specific source map errors, based on EventError
+"""
+
+
+class SourceMapError:
```

Comment:
I'd also ask around if `SourceMapError` is a good name here. Because in most cases, unless sourcemaps are invalid, or position for token is not found, it's not a source map issue really. It's more of a `SourceMapProcessingIssue` of sort, like we call it here https://github.com/getsentry/sentry/blob/master/src/sentry/models/processingissue.py

Reply:
@kamilogorek I can call it that, no problem